### PR TITLE
refactor: dogfood

### PR DESCRIPTION
**Pull request by bcollamore**

### CODE REVIEW

File: Philips.CodeAnalysis.MaintainabilityAnalyzers/Naming/EnforceBoolNamingConventionAnalyzer.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public class EnforceBoolNamingConventionAnalyzer : SingleDiagnosticAnalyzer
 		public EnforceBoolNamingConventionAnalyzer() : this(true, true) { }
 
 		public EnforceBoolNamingConventionAnalyzer(bool shouldCheckLocalVariables, bool shouldCheckFieldVariables)
-			: base(DiagnosticId.EnforceBoolNamingConvention, Title, MessageFormat, Description, Categories.Naming)
+			: base(DiagnosticId.EnforceBoolNamingConvention, Title, MessageFormat, Description, Categories.Naming, isEnabled: false)
```
Comment (ynsehoornenborg) [inline Philips.CodeAnalysis.MaintainabilityAnalyzers/Naming/EnforceBoolNamingConventionAnalyzer.cs:31]: Why disable this Analyzer ?

File: Philips.CodeAnalysis.MsTestAnalyzers/StringConstants.cs [hunk 2]
```diff
@@ -11,7 +11,9 @@ public static class StringConstants
 		public const string TestAttributeName = @"Test";
 		public const string EnsureAttributeName = @"Ensure";
 		public const string VerifyAttributeName = @"Verify";
-
+		public const string IsTrue = @"IsTrue";
```
Comment (ynsehoornenborg) [inline Philips.CodeAnalysis.MsTestAnalyzers/StringConstants.cs:14]: I would create more elaborate names, like IsTrueMethodName

**Review by ynsehoornenborg — APPROVED** [review]

File: Philips.CodeAnalysis.MaintainabilityAnalyzers/Naming/EnforceBoolNamingConventionAnalyzer.cs (hunk 1 above)
Comment (bcollamore) [reply Philips.CodeAnalysis.MaintainabilityAnalyzers/Naming/EnforceBoolNamingConventionAnalyzer.cs:31]: It was already disabled, and I accidentally enabled it in the last refactorings

File: Philips.CodeAnalysis.MsTestAnalyzers/StringConstants.cs (hunk 2 above)
Comment (bcollamore) [reply Philips.CodeAnalysis.MsTestAnalyzers/StringConstants.cs:14]: I don't like to put the intended use of the variable into variables names.  Is it IsTrueMethodName, IsTrueVariableName, IsTrueAssertMethodName? It artificially tries to dictate the usage.

**Pull request merged**